### PR TITLE
fixed the webpack dev server and importing of assets inside of an html

### PR DESCRIPTION
**Pull request by rfreitas**
1. HTML files are now loaded by the html loader, this allows to load assets from the html that go through other asset loaders
2. The image loader now stores the images in `dist/images`, url loader is no longer used in development, this brings production and development closer
3. The dev server is now serving code from the `dist` folder rather than the `src` folder, this also bring development and production closer and it fixes a bug where source maps were not working.

This is being used here: https://github.com/onfido/onfido-sdk-ui/blob/bank/demo/src/index.html#L95

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://87-pr-onfido-sdk-ui-onfido.surge.sh

File: webpack.config.babel.js [hunk 1]
```diff
@@ -91,7 +91,11 @@ const configDist = {
       },
       {
         test: /\.(svg|woff2?|ttf|eot|jpe?g|png|gif)(\?.*)?$/i,
-        loader: ENV==='production' ? 'file?name=[path][name]_[hash:base64:5].[ext]' : 'url'
+        loader: 'file?name=images/[name]_[hash:base64:5].[ext]'
+      },
+      {
+        test: /\.html$/,
+        loader: 'html-loader?interpolate'
```
Comment (stephencookdev) [inline webpack.config.babel.js]: Nice

File: package.json [hunk 2]
```diff
@@ -59,9 +59,11 @@
     "eslint-plugin-import": "^1.15.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.3.0",
+    "extract-loader": "0.0.2",
```
Comment (stephencookdev) [inline package.json]: What's this for? Or is it a peer dependency?

**Comment by rfreitas** [comment]
Travis automatic deployment: https://87-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**